app/src/main/java/edu/uc/zhukv/droneradarmap/DetailsFragment.kt

```diff
@@ -9,10 +9,6 @@ import android.view.ViewGroup
 
 class DetailsFragment : Fragment() {
 
-    companion object {
```

Comment:
I changed the order of the layout - https://kotlinlang.org/docs/coding-conventions.html#class-layout